graf3d/gl/inc/TGLUtil.h

```diff
@@ -246,11 +246,14 @@ inline void TGLVertex3::Set(const TGLVertex3 & other)
 class TGLVector3 : public TGLVertex3
 {
 public:
-   TGLVector3();
+   TGLVector3() = default;
    TGLVector3(Double_t x, Double_t y, Double_t z);
    TGLVector3(const Double_t *src);
-   TGLVector3(const TGLVector3 & other);
-   ~TGLVector3();
+   TGLVector3(const TGLVector3 &other);
+   ~TGLVector3() = default;
+
+   TGLVector3& operator = (const TGLVector3& v)
```

Comment:
I suspect that without copy ctor, the assignment can go away, too?

Reply:
In many cases one could remove copy constructor and assign operator - not only for these classes. Should we really do it for all classes?